### ISSUE TITLE
test upload fails if .app name contains a space

### ISSUE THREAD

**Issue by kamstrup** [Microsoft/mobile-center-cli#275]
A user has reported that mobile-center test fails upload of ios apps if the .app name contains a space.

**Comment by krukow** [comment]
Confirmed: App called "The App With Space" results in

```
 mobile-center test run xcuitest --app "My-Organization/App-With-Spaces" --devices "My-Organization/ios-10-x-smoke" --test-series "master" --locale "en_US" --build-dir ddp/Build/Products/Debug-iphoneos/
ditto: can't get real path for source 'ddp/Build/Products/Debug-iphoneos/The'

ditto: can't get real path for source 'App'
ditto: can't get real path for source 'With'
ditto: can't get real path for source 'Space.app'
ditto: can't get real path for source '/var/folders/hs/ycgt048163132zqg0mh_1r1h0000gn/T/ios-bundle-archiver117815-28454-ddtchg.rftmbtvs4i/Payload/The'
ditto: can't get real path for source 'App'
ditto: can't get real path for source 'With'

Error: Command 'test run xcuitest --app My-Organization/App-With-Spaces --devices My-Organization/ios-10-x-smoke --test-series master --locale en_US --build-dir ddp/Build/Products/Debug-iphoneos/' failed with exception "Cannot archive app bundle."
```
Fix will be planned

**Comment by RamonPage** [comment]
Same issue in Test Cloud _Classic_: https://github.com/xamarinhq/test-cloud-uploader/issues/45.

**Comment by RamonPage** [comment]
PR #279.

**Comment by RamonPage** [comment]
Fixed by #279.
